dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts

```diff
@@ -37,6 +38,10 @@ export const YesNoFormFieldProps = Object.assign(deepCopyObject(BaseFormFieldPro
     type: Boolean,
     default: false,
   },
+  evidenceDesired: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
I don't see the value of this new prop.
It is set to "true" for all "YesNoFormField" and "DataPointFormField" occurrences.
So why do we even need this as variable?

Reply:
For the DataPointFormField currently yes, but in the future there might be occurences which only have the value field and no further infos.
For YesNoFormFields there are occurences without evidenceRequired = true. For example in the lksg model. So the variable is not every time set to yes, but also takes on other values.

Reply:
Ok let's have a discussion after I have reviewed the latest state of this PR.
Maybe I will notice the necessity of `evidenceDesired` while doing that.

Reply:
Yes we will have the call today, since at some point today, I will have understood everything in the Files Changed.
Almost there.
(The evidenceDesired at least does not hurt.  So if big changes are needed, we could also do them later)

Reply:
I am able to discuss it now.

Reply:
Discussed with @stewal19 .    Stays as it is for now.  Closing